packages/audio-block/src/AudioBlock.tsx

```diff
@@ -138,7 +141,8 @@ export const AudioBlock = withAttachmentsProvider(({ appBridge }: BlockProps) =>
                     <div className="tw-flex-1">
                         <div data-test-id="block-title">
                             <RichTextEditor
-                                id={`${appBridge.getBlockId().toString()}-title`}
+                                key={titleKey}
+                                id={`${appBridge.context('blockId').get()}-title`}
```

Comment:
```suggestion
                                id={`${blockId}-title`}
```

Since we use it twice it would be nice to save it in a readable variable